event/src/unvalidated/signed/cacao.rs

```diff
@@ -37,6 +37,46 @@ pub struct Header {
     pub r#type: HeaderType,
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(transparent)]
+/// A wrapper around the a time value to hide the internal representation (which is currently a string).
+/// Use `From<chrono::DateTime<chrono::Utc>>` to construct if needed.
+pub struct CapabilityTime(String);
+
+impl From<chrono::DateTime<chrono::Utc>> for CapabilityTime {
+    fn from(time: chrono::DateTime<chrono::Utc>) -> Self {
+        Self(time.to_rfc3339_opts(chrono::SecondsFormat::AutoSi, true))
+    }
+}
+
+impl CapabilityTime {
+    /// Returns the time as a UTC DateTime
+    pub fn as_utc_dt(&self) -> anyhow::Result<chrono::DateTime<chrono::Utc>> {
+        let ts = Self::parse_timestamp(&self.0)
+            .map_err(|e| anyhow::anyhow!("invalid time format for '{}': {}", self.0, e))?;
+        Ok(ts.to_utc())
+    }
+
+    /// Returns a string representation of the time
+    pub fn as_str(&self) -> &str {
+        &self.0
+    }
+
+    fn parse_timestamp(input: &str) -> anyhow::Result<chrono::DateTime<chrono::Utc>> {
```

Comment:
do you think we should support try_into for DateTime to make this conversion easier/more native?

Reply:
yeah, probably nicer to do that. reworked in https://github.com/ceramicnetwork/rust-ceramic/pull/502/commits/7150a969eb4f97df82d0fdbed24f4582aec1667d